PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -3993,6 +3994,20 @@ void Finder::addFileNameTitle(const TCHAR * fileName)
 	_pMainMarkings->push_back(EmptySearchResultMarking);
 }
 
+void Finder::setLineNumFormatStr()
+{
+   size_t NumLines = (*_ppEditView)->execute(SCI_GETLINECOUNT);
+	size_t NumDigits = 0;

```

Comment:
I guess this shouldn't be indented this far?

Reply:
yes, you are right; I'll correct that together with the improvement for "Find in Files"